CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+## 0.1.2
+### Added
+- Use credentials from Docker config if none can be found otherwise ([]())
```

Comment:
This should be populated with the next PR that adds this functionality to `jib-maven-plugin`